Raven.Database/Storage/Esent/SchemaCreator.cs

```diff
@@ -1059,31 +1087,60 @@ internal void CreateIndexedDocumentsReferencesTable(JET_DBID dbid)
                 grbit = ColumnNotNullIfOnHigherThanWindowsXp()
             }, null, 0, out columnid);
 
-            CreateIndexes(tableid,
+            var by_id = "+id\0\0";
+            var by_key = "+key\0\0";
+            var by_view_and_key = "+view\0+key\0\0";
+            var by_ref = "+ref\0\0";
+
+            Api.JetCreateIndex2(session, tableid, new[]
+            {               
                 new JET_INDEXCREATE
                 {
                     szIndexName = "by_id",
-                    szKey = "+id\0\0",
-                    grbit = CreateIndexGrbit.IndexPrimary
-                },
-                new JET_INDEXCREATE
+                    cbKey = by_id.Length,
+                    szKey = by_id,
+                    grbit = CreateIndexGrbit.IndexDisallowNull | CreateIndexGrbit.IndexUnique,
```

Comment:
Same as above, `CreateIndexGrbit.IndexPrimary` -> `CreateIndexGrbit.IndexDisallowNull | CreateIndexGrbit.IndexUnique,` ?

Reply:
again not sure